mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/service/LoanApplicationWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -732,29 +740,66 @@ public CommandProcessingResult deleteApplication(final Long loanId) {
                 .build();
     }
 
+    public void validateMultiDisbursementData(final JsonCommand command, LocalDate expectedDisbursementDate){
+    	final String json = command.json();
+    	final JsonElement element = this.fromJsonHelper.parse(json);
+
+        final List<ApiParameterError> dataValidationErrors = new ArrayList<>();
+        final DataValidatorBuilder baseDataValidator = new DataValidatorBuilder(dataValidationErrors).resource("loan");
+        final JsonArray disbursementDataArray = command.arrayOfParameterNamed(LoanApiConstants.disbursementDataParameterName);
+        if(disbursementDataArray == null || disbursementDataArray.size() == 0){
```

Comment:
@chandrikamohith : It seems like a valid use case wherein no disbursement details are passed in the approval stage (since they are already present during the application stage)

This validation has been removed during merge, please let me know if it is going to cause an issue